### ISSUE TITLE
Error adding qscript as dependency 

### ISSUE THREAD

**Issue by poeticAndroid** [Nafees10/qscript#13]
It seems like the mere act of adding qscript to the package file causes an error..

in an empty folder I can run the following commands:
```
$ dub init
$ dub add qscript
$ dub
```
And I get the following errors:
```
Performing "debug" build using C:\D\dmd2\windows\bin\dmd.exe for x86.
utils 0.2.1: building configuration "library"...
C:\Users\poeticAndroid\AppData\Local\dub\packages\utils-0.2.1\utils\source\utils\lists.d(1182,19): Error: cannot implicitly convert expression this.size() - this.seek() > cast(ulong)n ? cast(ulong)n : this.size() - this.seek() of type ulong to uint
C:\D\dmd2\windows\bin\dmd.exe failed with exit code 1.
```

**Comment by Nafees10** [comment]
Sorry for the late reply, I'll have some free time today after 6-7 hours (have an exam starting in 2-3 hours), will fix it then.  
As a temporary fix, you could comment out the `FileReader` class in your `C:\Users\poeticAndroid\AppData\Local\dub\packages\utils-0.2.1\utils\source\utils\lists.d`.  
That class is not used anywhere in qscript, so it should be fine.

**Comment by Nafees10** [comment]
The issue was probably that you are on a 32 bit machine (arrays sizes are `uint` instead of `ulong`), and that `FileReader` in `utils` package used `ulong`.  
Its fixed in 0.6.3
